src/core-runtime.js

```diff
@@ -30,6 +30,14 @@ Globalize.locale = function( locale ) {
 	return this._locale;
 };
 
+Globalize._loadRuntimeFn = function( key ) {
+	var fn = Globalize[ key ];
+	if ( typeof fn === "function" ) {
+		fn.runtimeKey = key;
+	}
+	return fn;
+};
```

Comment:
Oh! For your proposed solution, we need this function that is similar to [common/runtime-bind.js](https://github.com/globalizejs/globalize/blob/master/src/common/runtime-bind.js). Can you move this into its own file in `common/` and update usage to look like [this](https://github.com/globalizejs/globalize/blob/master/src/date.js#L136) please (i.e., use a function directly instead of Globalize.<static method>)?

PS: It will be challenging to find a name to distinguish this from runtime-bind. Feel free to rename the latter to accommodate a good naming for both.